### DISCUSSION TITLE
BUG: "Empty result from content provider when adding event"

### DISCUSSION THREAD

**Discussion by keunes** [bitfireAT/icsx5#99] [General]
Hello,

For AntennaPod I've made an Events section on the website, with an ics file that contains all our one-off and recurring events. When trying to subscribe to the calendar I get a snackbar "Subscribed successfully". But then there's a red message "Empty result from content provider when adding event".

This is the URL of the problematic webcal file: https://antennapod.org/events-with-RDATE.ics

I suspect that the issue might be caused by the [RDATE](https://datatracker.ietf.org/doc/html/rfc5545#section-3.8.5.2) values we have in there. Other clients are having trouble with this as well.

Adding this same calendar as remote calendar in Thunderbird works just fine (RDATE values are respected).

If these aren't recognised, then ideally they're gracefully ignored, while the main recurrence is still there. Currently, no events are added.

ICSx5: F-Droid, v. 2.0.3-standard
OS: Android 11 (Fairphone 3)

**Comment by ArnyminerZ** [comment]
There are no issues for me, I can add the url to ICSx⁵, and the events show correctly on the Google Calendar app. Please, make sure you are running the latest version of the app. Otherwise, don't hesitate in writing back again and I'll help in everything I can.

**Reply by keunes** [reply]
Hi @ArnyminerZ, thanks for chipping in! TL;DR - can you please check/test with https://antennapod.org/events-with-RDATE.ics?

* As far as I can tell, I'm already on the latest version of ICSx⁵ (I added the version number to my OP after publishing).
* Why you weren't seeing the issue, is probably because we stripped out the `RDATE`s from the file by the time you checked it. Now our 'default' ICS/calendar file is 'failsafe' so that random visitors to our website get at least some data, but events that have been moved/manually added outside of the default recurrence will be missing for them.

**Reply by ArnyminerZ** [reply]
That makes sense, thank you. I will take a look asap

**Comment by ArnyminerZ** [comment]
Currently `RDATE PERIOD` is not supported by ical4android so neither by ICSx⁵. Please, keep track of [ical4android#31](https://github.com/bitfireAT/ical4android/issues/31), and feel free to propose a solution if you know how to.

This feature is not that well supported by Android, so it's pretty hard to implement. I suggest to use an alternative. Sorry for the inconvenience.
